src/framework/draw/bufferedpaintprovider.cpp

```diff
@@ -256,6 +256,11 @@ void BufferedPaintProvider::restore()
 {
 }
 
+double BufferedPaintProvider::deviceLogicalDpi() const
+{
+    return 0;
```

Comment:
Given that Painter divides by this value, `0` might not be the best dummy value (even while this class is currently not really used)